packages/mdc-top-app-bar/foundation.js

```diff
@@ -42,6 +46,10 @@ class MDCTopAppBarFoundation extends MDCFoundation {
       registerNavigationIconInteractionHandler: (/* type: string, handler: EventListener */) => {},
       deregisterNavigationIconInteractionHandler: (/* type: string, handler: EventListener */) => {},
       notifyNavigationIconClicked: () => {},
+      registerScrollHandler: (/* handler: EventListener */) => {},
+      deregisterScrollHandler: (/* handler: EventListener */) => {},
+      getViewportScrollY: () => /* number */ 0,
+      totalActionItems: () => /* number */ 0,
```

Comment:
Sorry I realized this late, but this probably deserves to be called `getTotalActionItems` (verbs in function names are good)